packages/terra-core-docs/src/terra-dev-site/doc/form-radio/example/field/ControlledRadioField.jsx

```diff
@@ -20,6 +20,7 @@ export default class extends React.Component {
   }
 
   handleOnChange(e) {
+    console.log(e.currentTarget.value);
```

Comment:
please remove this

Reply:
removed https://github.com/cerner/terra-core/pull/3868/commits/84b1bdffbf48a97543ede12eebbfedba7a355604